File: data_fill/scirpts/task_result.py
import random
import string

w = open('task_result.js', 'w', encoding="utf-16")

template = """db.task_result.create(
{{
    grade: {0},
    studentId: {1},
    taskId: {2}
}});
"""

task = [1, 2]
module = 0

for i in range(len(module_st)):
    for j in range(random.randint(3, 7)):
        studentId = j + 1 + module
        taskId = task[i]
        grade = random.randint(3, 5)

        out = template.format(
                grade, studentId, taskId)
        w.write(out)

    module = module + module_st[i]

File: data_fill/task_result.js
var db = require('../models');

module.exports.Create = () =>
{
    db.task_result.create(
    {
        grade: 5,
        studentId: 22,
        taskId: 1
    });
    db.task_result.create(
    {
        grade: 3,
        studentId: 2,
        taskId: 1,
        comment: "Неверно выполнено задание"
    });
    db.task_result.create(
    {
        grade: 4,
        studentId: 3,
        taskId: 1
    });
    db.task_result.create(
    {
        grade: 5,
        studentId: 4,
        taskId: 1
    });
    db.task_result.create(
    {
        grade: 5,
        studentId: 5,
        taskId: 1
    });
    db.task_result.create(
    {
        grade: 5,
        studentId: 6,
        taskId: 1
    });
    db.task_result.create(
    {
        grade: 4,
        studentId: 1,
        taskId: 2
    });
    db.task_result.create(
    {
        grade: 4,
        studentId: 2,
        taskId: 2
    });
    db.task_result.create(
    {
        grade: 5,
        studentId: 3,
        taskId: 2
    });
    db.task_result.create(
    {
        grade: 4,
        studentId: 4,
        taskId: 2
    });
    db.task_result.create(
    {
        grade: 3,
        studentId: 5,
        taskId: 2
    });
}